### ISSUE TITLE
Error During Installation

### ISSUE THREAD

**Issue by mcgaritydotme** [benknight/hue-alfred-workflow#58]
Using Alfred 4.5, I receive the following error when attempting to connect to my bridge (either using the manual button or explicitly entering the bridge's IP address):

`
{
  "items": [
    {
      "valid": false,
      "subtitle": "u'username'",
      "icon": {
        "path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertStopIcon.icns"
      },
      "title": "Error in workflow 'Philips Hue Controller'"
    }
  ]
}
`

**Comment by mjl** [comment]
I just had the same experience and tried debugging.

It seems action.py:209 should read

```python
if this_query.startswith('set_bridge:'):
```

or perhaps

```python
if query[0] == 'set_bridge':
```

otherwise it will never call the setup function.

**Comment by benknight** [comment]
Can you guys try downloading the version from the releases page while I
debug this? sorry about that.

**Comment by benknight** [comment]
The last version*

**Comment by mcgaritydotme** [comment]
Thanks for the responses.  I got this error using the latest version from the downloads page (3.0.5, from July)

**Comment by benknight** [comment]
There was a logic bug introduced in #57.  I've fixed and published as v3.0.6

**Comment by mcgaritydotme** [comment]
Tested successfully.  Thank you for the turnaround, and great work on this overall!
